File: M.I.L.K./ImageCollectionViewCell.m
//
//  ImageCollectionViewCell.m
//  M.I.L.K.
//
//  Created by noah davidson on 12/12/17.
//  Copyright © 2017 noah davidson. All rights reserved.
//

#import "ImageCollectionViewCell.h"

@implementation ImageCollectionViewCell

- (void)prepareForReuse
{
    [super prepareForReuse];
    self.imageView.image = nil;
}

@end

File: M.I.L.k/ImageCollectionViewCell.swift
//
//  ImageCollectionViewCell.swift
//  M.I.L.k
//
//  Created by noah davidson on 12/13/17.
//  Copyright © 2017 noah davidson. All rights reserved.
//

import UIKit

class ImageCollectionViewCell: UICollectionViewCell {
    
    @IBOutlet weak var imageView: UIImageView!
    
    override func prepareForReuse() {
        super.prepareForReuse()
        imageView.image = nil
    }
}
